net/relayd/src/opnsense/mvc/app/models/OPNsense/Relayd/Relayd.xml

```diff
@@ -0,0 +1,326 @@
+<model>
+   <mount>//OPNsense/relayd</mount>
```

Comment:
yes, what I thought earlier, it's all in one model. You can basically keep this layout as is, but split it up, increasing the mount point //OPNsense/relayd/general  etc.

Reply:
I'd rather avoid that. It produces redundant code.

Reply:
Ok, the CRUD thing should help to reduce the code size in the existing controller now anyway :)